packages/create-fuels/src/lib/setupProgram.ts

```diff
@@ -19,9 +19,9 @@ export const setupProgram = () => {
     .option('-c, --contract', 'Include contract program')
     .option('-p, --predicate', 'Include predicate program')
     .option('-s, --script', 'Include script program')
```

Comment:
```suggestion
```

Given that we're removing the prompt, should we remove the cli options as well?

Reply:
@petertonysmith94 I tried doing that, but couldn't find a way. Do you know of a solution?

Reply:
@petertonysmith94 I checked this already, and it's not that. Even if someone is using pnpm or bun, I've found `process.argv[0]` to always be the path to the `node.exe` file on the user's machine. Not pnpm, or bun.